test/src/vfs_helpers.h

```diff
@@ -594,6 +594,24 @@ struct TemporaryDirectoryFixture {
     return array_uri;
   };
 
+  /**
```

Comment:
Should these live in another helper file than vfs_helpers?

Reply:
These are member functions of the `TemporaryDirectoryFixture`. They require the `ctx` member to run.